stan/math/prim/scal/meta/scalar_seq_view_writable.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef STAN_MATH_PRIM_SCAL_META_SCALAR_SEQ_VIEW_WRITABLE_HPP
+#define STAN_MATH_PRIM_SCAL_META_SCALAR_SEQ_VIEW_WRITABLE_HPP
+
+#include <stan/math/prim/scal/meta/scalar_type.hpp>
+
+namespace stan {
+  /**
+   * scalar_seq_view_writable provides a uniform sequence-like wrapper around
+   * either a scalar or a sequence of scalars. It performs exactly the same as
+   * scalar_seq_view but without the const variables
+   *
+   * @tparam C the container type; will be the scalar type if wrapping a scalar
+   * @tparam T the scalar type
+   */
+  template <typename C, typename T = typename scalar_type<C>::type>
```

Comment:
Nope! I'm wrong. I see how VectorBuilder works now. It can be std::vector or scalar. I think this can work. I will try it.